sdk/src/main/scala/com/horizen/SidechainState.scala

```diff
@@ -196,29 +196,13 @@ class SidechainState private[horizen] (stateStorage: SidechainStateStorage,
     })
   }
 
-  private def getOrElseWithdrawalEpochInfo(): WithdrawalEpochInfo = {
-    stateStorage.getWithdrawalEpochInfo.getOrElse(WithdrawalEpochInfo(0,0))
-  }
-
-  private def getAllowedWithdrawalRequestBoxesPerBlock: Int = {
-    params.maxWBsAllowed / (params.withdrawalEpochLength - 1)
-  }
-
   def getAlreadyMinedWithdrawalRequestBoxesInCurrentEpoch: Int = {
-    stateStorage.getWithdrawalRequests(getOrElseWithdrawalEpochInfo().epoch).size
+    stateStorage.getWithdrawalRequests(stateStorage.getWithdrawalEpochInfo.getOrElse(WithdrawalEpochInfo(0,0)).epoch).size
```

Comment:
let's use `getWithdrawalEpochInfo()` instead of `stateStorage.getWithdrawalEpochInfo.getOrElse(WithdrawalEpochInfo(0,0)` to avoid code duplication.

Reply:
Done